src/components/FlowsTable.vue

```diff
@@ -197,7 +188,7 @@
 }
 
 .flows-table__activity-chart { @apply
-  h-12
+  !h-12
```

Comment:
Seems suspicious. Why the `!`?

Reply:
Honestly might not be needed, the chart component doesn't do a good job growing to allowed space so this was a way to force it to do so. Maybe higher CSS specificity is needed?

Reply:
Not a big deal. But if there's a specificity issue that would be something to fix possibly. We shouldn't have to worry about specificity especially for something like height.

Reply:
Agreed _usually_; these canvas and svg charts are a little weird when it comes to responsiveness